src/libraries/Native/Unix/System.Native/pal_networkchange.c

```diff
@@ -162,10 +162,8 @@ void SystemNative_ReadEvents(int32_t sock, NetworkChangeEvent onNetworkChange)
             case RTM_ADD:
             case RTM_DELETE:
             case RTM_REDIRECT:
-            {
```

Comment:
This doesn't seem to be related to the rest of the PR. I'd prefer not making an unrelated change.

Reply:
I noted it in the description above. It was a very minor nit (extra brace) from the PR #1602, so i added here. Is it ok to keep it, since we have a ref? :)

Reply:
I would rather remove it and fix it separately.